diff -ruN a/museeq/chatrooms.cpp b/museeq/chatrooms.cpp
--- a/museeq/chatrooms.cpp	2017-01-28 10:47:25.000000000 +0000
+++ b/museeq/chatrooms.cpp	2017-01-28 17:17:49.361459329 +0000
@@ -75,7 +75,7 @@
 		}
 	}
 
-	ChatRoom* _room = new ChatRoom(room, this, false);
+	ChatRoom* _room = new ChatRoom(room, this, (const char*)0);
 	addTab(_room, room);
 	setCurrentWidget(_room);
 	connect(_room, SIGNAL(highlight(int, QWidget*)), this, SIGNAL(highlight(int)));
diff -ruN a/museeq/searches.cpp b/museeq/searches.cpp
--- a/museeq/searches.cpp	2017-01-28 10:47:25.000000000 +0000
+++ b/museeq/searches.cpp	2017-01-28 17:18:49.314793178 +0000
@@ -158,7 +158,7 @@
 
     // Create the tab if it's a wishlist result
 	if ((i == mSearchTabWidget->count()) && museeq->isInWishlist(query)) {
-		Search* s = new Search(query, mSearchTabWidget, false);
+		Search* s = new Search(query, mSearchTabWidget, (const char*) 0);
 		mSearchTabWidget->addTab(s, query);
 
         // Highlight the search icon so that the user know that there are some new search results
@@ -194,7 +194,7 @@
 		}
 	}
 	if(i == mSearchTabWidget->count()) {
-		Search* s = new Search(q, mSearchTabWidget, false);
+		Search* s = new Search(q, mSearchTabWidget, (const char*) 0);
 		mSearchTabWidget->addTab(s, q);
 
         // Highlight the search icon so that the user know that there are some new search results
